### ISSUE TITLE
Colab Install Script syntax error [BUG]

### ISSUE THREAD

**Issue by gumdropsteve** [rapidsai/notebooks-contrib#149]
### Describe the bug
RAPIDS Colab install script syntax error in line 7 

### Steps/Code to reproduce bug
- Inside Google Colab Tesla T4 session 
  - Click [here](https://colab.research.google.com/drive/1SuHxJA1G-jw4J9gMZ3oqL1Ck0O2ZIKp1#forceEdit=true&offline=true&sandboxMode=true) for such a session with input cell ready to reproduce error

**input**
```
!wget -nc https://github.com/rapidsai/notebooks-contrib/blob/master/utils/rapids-colab.sh
!bash rapids-colab.sh

import sys, os

sys.path.append('/usr/local/lib/python3.6/site-packages/')
os.environ['NUMBAPRO_NVVM'] = '/usr/local/cuda/nvvm/lib64/libnvvm.so'
os.environ['NUMBAPRO_LIBDEVICE'] = '/usr/local/cuda/nvvm/libdevice/'
```
**output**
```
--2019-08-15 21:35:17--  https://github.com/rapidsai/notebooks-contrib/blob/master/utils/rapids-colab.sh
Resolving github.com (github.com)... 140.82.113.4
Connecting to github.com (github.com)|140.82.113.4|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: ‘rapids-colab.sh’

rapids-colab.sh         [ <=>                ]  74.83K  --.-KB/s    in 0.05s   

2019-08-15 21:35:17 (1.45 MB/s) - ‘rapids-colab.sh’ saved [76630]

rapids-colab.sh: line 7: syntax error near unexpected token `newline'
rapids-colab.sh: line 7: `<!DOCTYPE html>'
```

### Expected behavior
Script should be pulled from `notebooks-contrib` repo and run in Colab
  - installing RAPIDS AI suite and setting up environment for use of RAPIDS

### Environment details (please complete the following information):
- Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
  - Google Colaboratory
  - Hardware accelerator
    - GPU: Tesla T4 
- Method of RAPIDS libraries install: [conda, Docker, or from source]
  - Not applicable to issue 
    - Execution halts before command is run
  - If other methods, provide the commands used
    - `rapids-colab.sh`
      - `!wget -nc`
      - `!bash`
 
### Additional context
- Have been having issues properly installing RAPIDS suite for past few days
  - E.g. `TypeError: read_csv() got an unexpected keyword argument 'parse_dates'` where `parse_dates` is a keyword argument 
  - Number of updates to cuDF have been pushed in the same time window as this issue's arising
    - Switching the install script from `nightly` to a `stable` version may help resolve

**Comment by gumdropsteve** [comment]
More context: [here](https://github.com/rapidsai/notebooks-contrib/blob/master/utils/rapids-colab.sh) is the link to rapids-colab.sh (the script colab is saying has a syntax error & being referenced in this issue)

**Comment by gumdropsteve** [comment]
resolved by updated `rapids-colab.sh` merged from pr #159

**Comment by lbotecur** [comment]
**Trying to run the notebook in [https://rapids.ai/start.html](https://rapids.ai/start.html) (button LAUNCH IN COLAB), when I run this cell:**

`# Install RAPIDS`
!git clone https://github.com/rapidsai/rapidsai-csp-utils.git
!bash rapidsai-csp-utils/colab/rapids-colab.sh

import sys, os

dist_package_index = sys.path.index('/usr/local/lib/python3.6/dist-packages')
sys.path = sys.path[:dist_package_index] + ['/usr/local/lib/python3.6/site-packages'] + sys.path[dist_package_index:]
sys.path
exec(open('rapidsai-csp-utils/colab/update_modules.py').read(), globals())

**I get this error:**

Cloning into 'rapidsai-csp-utils'...
remote: Enumerating objects: 25, done.
remote: Counting objects: 100% (25/25), done.
remote: Compressing objects: 100% (24/24), done.
remote: Total 25 (delta 5), reused 5 (delta 0), pack-reused 0
Unpacking objects: 100% (25/25), done.
PLEASE READ
`********************************************************************************************************`
Colab Notebooks Migration Notice:
 
We have changed the location of the Colab script to our new CSP utilties repo.
We are also dropping support for versions below 0.11, as we are on 0.13 nightlies
 
Not all Colab notebooks are updated (like personal Colabs) and/or are using outdated versions of RAPIDS,
so we HIGHLY encourage users to update their scripts and Colab notebooks both with the new link AND with 
to the new API.  Otherwise, your code MAY BREAK
 
This notice will disappear/be updated on our next release.
 
Please enter in the box your desired RAPIDS version (ex: '0.11' or '0.12', between 0.11 to 0.13, without the quotes)
and hit Enter. If you need stability, use 0.12. If you want bleeding edge, use our nightlies (0.13), but things can break.
0.11
RAPIDS Version modified to 0.11 stable
rapidsai-csp-utils/colab/rapids-colab.sh: line 70: syntax error near unexpected token 'else'
`***********************************************************************`
Let us check on those pyarrow and cffi versions...
`***********************************************************************`

unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
unloaded pyarrow 0.14.1
loaded pyarrow 0.14.1
You're now running pyarrow 0.14.1 and are good to go!
unloaded cffi 1.14.0
loaded cffi 1.14.0
